app/src/main/res/values-bg/strings.xml

```diff
@@ -0,0 +1,746 @@
+        <item>"Плуване"</item>
+        <item>"Планинско тичане"</item>
+        <item>"Воден джогинг"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_beans">
+        <item>"¼ чаша хумус или бобова разядка"</item>,
+        <item>"½ чаша сварен боб, грах, леща, тофу или темпе"</item>,
+        <item>"1 чаша пресен грах или покълнала леща"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_berries">
+        <item>"½ чаша пресни или замразени боровинки"</item>
+        <item>"¼ чаша изсушени боровинки"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_other_fruits">
+        <item>"1 средно голям плод"</item>
+        <item>"1 чаша нарязани плодове"</item>
+        <item>"¼ чаша сушени плодове"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_cruciferous_vegetables">
+        <item>"½ чаша нарязани цветнолистни"</item>
+        <item>"¼ чаша брюкселско зеле или броколи"</item>
+        <item>"1 супена лъжица хрян"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_greens">
+        <item>"1 чаша сурови зеленолистни"</item>
+        <item>"½ чаша варени зеленолистни"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_other_vegetables">
+        <item>"1 чаша сурови листни зеленчуци"</item>
+        <item>"½ чаша сурови или варени зеленчуци"</item>
+        <item>"½ чаша зеленчуков сок"</item>
+        <item>"¼ чаша сушени гъби"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_flaxseeds">
+        <item>"1 супена лъжица ленено семе"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_nuts">
+        <item>"¼ чаша ядки или семена"</item>
+        <item>"2 супени лъжици ядки или масло от семена"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_spices">
+        <item>"¼ чаена лъжичка куркума"</item>
+        <item>"Всякакви други безсолни билки и подправки, които обичате"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_whole_grains">
+        <item>"½ чаша топла зърнена закуска или сварени зърна, паста или царевица"</item>
+        <item>"1 чаша студена зърнена закуска"</item>
+        <item>"1 тортила или филия хляб"</item>
+        <item>"½ поничка или английски мъфин"</item>
+        <item>"3 чаши пуканки"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_beverages">
+        <item>"Една чаша (350 мл.)"</item>
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_exercise">
+        <item>"90 минути умерена интензивност на физическа дейност"</item>
+        <item>"40 минути енергично интензивност дейност"</item>
+    </string-array>
+
+    <string name="error_cannot_handle_url">"Връзката не може да се отвори. Моля, инсталирайте браузър."</string>
+
+    <string-array name="food_videos_beans" translatable="false">
```

Comment:
Please revert all of these food_videos_* string-arrays back to English. Anything marked with translatable="false" should not be translated and should be left in English.